### PR TITLE
Adding a new example to create a Dns record to add a custom domain and purchase a Managed Certificate (free) and secure the web app asite

### PR DESCRIPTION
**Pull request by coelho5br**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [ X ] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

*
*
*

### CODE REVIEW

**Comment by bmoore-msft** [comment]
@coelho5br - just looking at the template I think we may be able to automate it... but at a minimum will need step-by-step docs in the readme in order to be able to manually validate it.  So if you want to start with the steps that will help figure out if we can automate it

File: quickstarts/microsoft.web/asp-webapp-customdomain-asmc-certificate/README.md [hunk 1]
```diff
@@ -18,11 +18,13 @@ To deploy this template, you need to have the following resources:
...
 3. A domain purchased that has a DNS zone hosted at Azure DNS. 
    You can confirm the Name Server records (NS type) querying the domain at https://digwebinterface.com
+
+   This is because the template will create the required DNS records to Bind the custom domain and validate the Manged Certificate ownership.
 4. **Make sure you Web App does not have any Network restriction or redirection (such as http to https or to another url). When generating certificate to the apex/root domain**
 
 This arm deployment will:
 
-1. Createa a DNS record with ASUID, CNAME or A to your Azure DNS zone responsible to resolve names at this domain. This will be used to validate your custom domain while adding to the Web App.
+1. Create DNS record with ASUID, CNAME or A at your Azure DNS zone responsible to resolve names at this domain. This will be used to validate your custom domain while adding to the Web App.
```
Comment (bmoore-msft) [inline quickstarts/microsoft.web/asp-webapp-customdomain-asmc-certificate/README.md:27]: The error message in deployment is looking for a TXT record, not CNAME/A

Comment (coelho5br) [reply quickstarts/microsoft.web/asp-webapp-customdomain-asmc-certificate/README.md:34]: @bmoore-msft 
The error still related to azure DNS not been the authoritative for that domain.
Any luck testing with the DNS I shared?

Comment (coelho5br) [reply quickstarts/microsoft.web/asp-webapp-customdomain-asmc-certificate/README.md:34]: @bmoore-msft 
Any news?

**Pull request closed without merging**